tasks/delete_record.json

```diff
@@ -25,6 +26,10 @@
         "instance": {
             "description": "ServiceNow instance name. If your instance uri is https://dev84270.service-now.com your instance name would be dev84270.",
             "type": "Optional[String[1]]"
+        },
+        "access_token": {
+            "description": "ServiceNow OAuth access token. Use the get_token task to generate one.",
```

Comment:
Should probably update this to remove mention of `get_token`. Think the other metadata.json files need the update too?